src/components/pages/JobDetails.vue

```diff
@@ -196,6 +196,13 @@ export default {
                 <div class="widget">
                   <div class="job-overview">
                     <ul>
+                      <li>
```

Comment:
Can we add this section to the bottom of the list? This is not-so-important information, let's not show this at the top of the widget?

Reply:
Actually, we can merge this as is. I just took a quick look on my local, it does look good. I will just change the date formatting.

`new Date(postData.updated_at).toLocaleDateString()` returns `8/15/2019` which is in US format.

`new Date(postData.updated_at).toLocaleDateString('tr-TR', { year: 'numeric', month: 'long', day: 'numeric' })` returns `15 Ağustos 2019` which is much better.